win32/build/confutils.js

```diff
@@ -2096,7 +2096,9 @@ function generate_files()
 
 	if (INVALID_CONFIG_ARGS.length) {
 		STDOUT.WriteLine('WARNING');
-		STDOUT.WriteLine('The following arguments is invalid, and therefore ignored:');
+		STDOUT.WriteLine('The following ' +
+			INVALID_CONFIG_ARGS.length > 1 ? 'arguments are' : 'argument is' +
```

Comment:
```suggestion
			(INVALID_CONFIG_ARGS.length > 1 ? 'arguments are' : 'argument is') +
```

It seems that these blocks need to be also wrapped in the parentheses. Then it will work ok.